src/devices_models/device_constructors/hydrogeneration_constructor.jl

```diff
@@ -41,7 +41,6 @@ function construct_device!(psi_container::PSIContainer, sys::PSY.System,
     reactivepower_variables!(psi_container, devices);
 
     #Constraints
-    activepower_constraints!(psi_container, devices, model, S, model.feed_forward)
```

Comment:
Wait, this doesn't make any sense. Why active power constraints are eliminated a Power Plant doesn't have unlimited power capacity.

Reply:
well because for `HydroReserviorFlow`  has sum( P) <= limit constraint. So you dont want to add another P <= Forecast constraint.

Reply:
I agree you don't want P <= Forecast. This has to be P <= P_max

Reply:
When the variables are created the (Pmin, Pmax) constraint are added.

Reply:
With these changes: 
![image](https://user-images.githubusercontent.com/7231396/73233628-4df98080-4144-11ea-96c9-81f96742ac7a.png)

Previously:
![image](https://user-images.githubusercontent.com/7231396/73233939-51d9d280-4145-11ea-87a9-7bc883caee5d.png)